src/MongoDB/Cursor.c

```diff
@@ -351,6 +351,21 @@ PHP_METHOD(Cursor, rewind)
 }
 
 /* {{{ MongoDB\Driver\Cursor function entries */
+ZEND_BEGIN_ARG_WITH_TENTATIVE_RETURN_TYPE_INFO_EX(ai_Cursor_current, 0, 0, IS_MIXED, 0)
+ZEND_END_ARG_INFO()
+
+ZEND_BEGIN_ARG_WITH_TENTATIVE_RETURN_TYPE_INFO_EX(ai_Cursor_key, 0, 0, IS_LONG, 0)
+ZEND_END_ARG_INFO()
+
+ZEND_BEGIN_ARG_WITH_TENTATIVE_RETURN_TYPE_INFO_EX(ai_Cursor_next, 0, 0, IS_VOID, 0)
+ZEND_END_ARG_INFO()
+
+ZEND_BEGIN_ARG_WITH_TENTATIVE_RETURN_TYPE_INFO_EX(ai_Cursor_valid, 0, 0, _IS_BOOL, 0)
+ZEND_END_ARG_INFO()
+
+ZEND_BEGIN_ARG_WITH_TENTATIVE_RETURN_TYPE_INFO_EX(ai_Cursor_rewind, 0, 0, IS_VOID, 0)
+ZEND_END_ARG_INFO()
+
```

Comment:
Same as with `JsonSerializable`, these could be defined in a central place, but for iterator methods I'm not sure they're needed in many other places.

Reply:
The Iterator and Countable methods are definitely not reused, so I think this is fine as-is. [PHPC-1938](https://jira.mongodb.org/browse/PHPC-1938) will definitely help to keep this more organized.